boards/lilygo/ttgo_t7v1_5/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 Tobias Kässer <t.kaesser@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config HEAP_MEM_POOL_ADD_SIZE_BOARD
+	int
+	default 65535 if WIFI && BT
```

Comment:
@kaessert, let this be only `default 4096` instead. You can check other esp32-based boards as reference.

Reply:
done